dallinger/recruiters.py

```diff
@@ -844,6 +853,39 @@ def get_qualifications(self, hit_id, sandbox):
             "peripheral_requirements": details["peripheral_requirements"],
         }
 
+    def screen_out(self, assignment_id: str):
```

Comment:
The method signature doesn't match that of the superclass; it should accept `participant`, not `assignment_id`. Also, I think that we should make `bonus` an argument instead of inferring it from the participant.

Reply:
Method signature adapted